src/Network/Network/Models/AzureFirewall/PSAzureFirewallHubIpAddresses.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.Azure.Commands.Network.Models
 {
     public class PSAzureFirewallHubIpAddresses
     {
-        public PSAzureFirewallPublicIpAddress[] publicIPAddresses { get; set; }
```

Comment:
Hi @Gizachew-Eshetie removing a public property is a breaking change and has been detected by CI.

Reply:
@isra-fel suppressed the breaking change static analysis error

Reply:
@Gizachew-Eshetie that's not what I meant. Only false positives can be suppressed. For real breaking changes:
1. We will defer the merging of this PR to next release
2. To preannounce the breaking change to our customers, please submit another PR to add [breaking change attribute](https://github.com/Azure/azure-powershell/blob/main/documentation/breaking-changes/breaking-changes-attribute-help.md) to the cmdlet.

Reply:
@isra-fel, can I add breaking change attribute to the cmdlet in this PR and merge this PR in September breaking changes sprint or should I have to create a separate PR to add breaking changes attribute? If I have to create a separate PR, do I have to include all commits to this PR or just only a commit that has breaking change attribute? Also I revert the breaking suppression back.

Reply:
In short, please create a separate PR which only adds breaking change attributes (and changelog).
The point is, breaking change attributes should be checked in in September release because the purpose is to preannounce; but the actual breaking changes should be checked in to October release, which would be a major release that allows breaking changes. So there's got to be 2 PRs.

Reply:
@isra-fel created [this PR](https://github.com/Azure/azure-powershell/pull/19363) to preannounce. Please let me know if I miss anything.